modules/redisearch.ts

```diff
@@ -890,7 +1084,7 @@ export type FTIndexType = 'HASH' | 'JSON';
 /**
  * The config response
  */
-export type FTConfig = {
+export interface FTConfig {
```

Comment:
What's the benefit of this being an interface rather than a type?

Reply:
https://github.com/danitseitlin/redis-modules-sdk-ts/pull/136#discussion_r698264723
The same as the jsdocs, IDEs can show interface names in documentation hints while they usually can't show types